doc/Extensions/Applications.md

```diff
@@ -2759,6 +2759,11 @@ b.) "public_key_to_arbitrary_name" - A dictionary to convert between the publick
 
 5. Restart snmpd.
 
+On Ubuntu, snmpd runs as Debian-snmp, and as a result cannot "directly" access or run wg. BUT - if you add this user (Debian-snmp) to visudo, and allow `bash` to be run, then you can update the entry in snmpd.conf to be,
```

Comment:
That is a bigger security risk than the one already documented in the top of the file.

Reply:
Understand, but without this - it won't run / work on Ubuntu 😉 

Thanks!

Reply:
Would be much better if it could be restricted to "wg show all dump"